portal/server/routes/middleware/validatePartialAuthToken/index.test.js

```diff
@@ -62,15 +61,32 @@ describe('validatePartialAuthToken', () => {
 
     describe('when portal api fails to validate the token as a partial 2fa token', () => {
       beforeEach(() => {
-        when(api.validatePartialAuthToken)
-          .calledWith(userToken)
-          .mockRejectedValueOnce(new Error('test error'));
+        when(api.validatePartialAuthToken).calledWith(userToken).mockRejectedValueOnce(new Error('test error'));
       });
 
       itDoesNotCallNext();
       itDestroysTheSession();
       itRedirectsToLoginPage();
     });
+
+    describe('when portal api successfully validates the token as a login complete', () => {
```

Comment:
Should we add assertions where we check following 2FA specific properties values:

* signInLinkSendCount
* signInToken